src/status_im/transport/filters.cljs

```diff
@@ -72,7 +74,10 @@
 
 (re-frame/reg-fx
  :shh/remove-filters
- (fn [filters]
+ (fn [[filters callback]]
    (doseq [[chat-id filter] filters]
-     (when filter (remove-filter! {:chat-id chat-id
-                                   :filter filter})))))
+     (when filter (remove-filter!
+                   {:chat-id          chat-id
+                    :filter           filter
+                    :success-callback false})))
+   (callback)))
```

Comment:
yep, that's most likely the case

Reply:
i would polish this in a separate PR as it most likely wouldn't fix the current issue because of the queue of polling requests which still can cause RPC call after `StopNode`, just because we have too many requests when account contains, let's say, 100 chats.  On the other hand, it can make logging out much slower on android devices, for instance, so better to work on it separately.